x-pack/plugins/canvas/server/routes/workpad.js

```diff
@@ -139,6 +188,30 @@ export function workpad(server) {
     },
   });
 
+  // update workpad assets
+  server.route({
+    method: 'PUT',
+    path: `${routePrefixAssets}/{id}`,
+    config: { payload: { allow: 'application/json', maxBytes: 26214400 } }, // 25MB payload limit
+    handler: function(request) {
+      return updateWorkpadAssets(request)
+        .then(() => ({ ok: true }))
+        .catch(formatResponse);
+    },
+  });
+
+  // update workpad structures
+  server.route({
+    method: 'PUT',
+    path: `${routePrefixStructures}/{id}`,
+    config: { payload: { allow: 'application/json', maxBytes: 26214400 } }, // 25MB payload limit
```

Comment:
I don't think the workpad payload itself needs to have a 25MB limit, just the entire document (and it's mostly made up of the assets). It doesn't matter if we leave this here though, it shouldn't affect anything.